Project/Project/Pages/Index.cshtml

```diff
@@ -4,15 +4,32 @@
     ViewData["Title"] = "Home page";
 }
 
-<h1>Know Your Favorite Nobel Laureate</h1>
+<style>
+    table {
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+    }
+
+    td, th {
+
+        text-align: left;
+        padding: 8px;
+    }
+
+    tr:nth-child(even) {
+        background-color:lightgrey;
+    }
+</style>
```

Comment:
Embedded styles are not XHTML 1.0 Strict compliant.  For XHTML 1.0 Strict compliance, styles should not be in HTML markup, but should be in linked stylesheets.